ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -158,8 +163,8 @@ boolean isValidPlacement(String placementId) {
    */
   private void cleanLeakedBannerAdapters() {
     for (String placementId : new HashSet<>(mVungleBanners.keySet())) {
-      VungleBannerAd bannerAd = mVungleBanners.get(placementId);
-      if (bannerAd != null && bannerAd.getAdapter() == null) {
+      VungleBannerAdapter bannerAd = mVungleBanners.get(placementId);
```

Comment:
this seems to remove a workaround for leaking VungleInterstialAdapter(which holds a strong ref to VungleBannerAdapter and "bannerAd.getAdapter()" was a check to checks if VungleInterstialAdapter was GCetd)
please revert. If mVungleBanners contains VungleBannerAdapter, and VungleInterstialAdapter is GCted we can't figure it out.
@HuFangshuai

Reply:
Yes. Hi @felixzhang0703 here's the related PR https://github.com/googleads/googleads-mobile-android-mediation/pull/304